### PR TITLE
Changes in social media icons, select network and wallet fields

### PR DESCRIPTION
**Pull request by Kimi82**

### CODE REVIEW

File: src/static/icons.ts [hunk 1]
```diff
@@ -46,7 +47,9 @@ const icons: { [key: string]: string } = {
   phantom: phantomIcon,
   sollet: solletIcon,
   'math wallet': mathIcon,
-  clear: clearIcon
+  clear: clearIcon,
+  Testnet: mainnet,
```
Comment (awojciak) [inline src/static/icons.ts:51]: Mainnet

File: src/static/icons.ts [hunk 2]
```diff
@@ -46,7 +47,9 @@ const icons: { [key: string]: string } = {
...
+  Localnet: devnet
```
Comment (awojciak) [inline src/static/icons.ts:52]: Devnet

**Review by awojciak — COMMENTED** [review]
adjust hamburger menu to mockups

**Review by awojciak — COMMENTED** [review]
obraz
I think bottom padding of this modal should be smaller

**Comment by Kimi82** [comment]
> adjust hamburger menu to mockups

You mean, i need to change mobile hamburger menu?
If so, I haven't done it yet because I can't find the changes in the mockups :/

**Comment by awojciak** [comment]
> > adjust hamburger menu to mockups
> 
> You mean, i need to change mobile hamburger menu? If so, I haven't done it yet because I can't find the changes in the mockups :/

obraz

**Comment by PawelKrawczyk96** [comment]
On figma icon and Server name is horizontally centered, on project we have alignment to bottom.
Screenshot from 2021-10-04 09-05-40

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Add details also to Synthetics

**Review by awojciak — APPROVED** [review]

**Pull request merged**